scripts/validate-openapi.js

```
@@ -0,0 +1,237 @@
+    if (!validMethods.includes(method.toLowerCase())) {
+      console.warn(`⚠️  Warning: Invalid HTTP method '${method}' in path '${path}'`);
+      issues++;
+    }
+  });
+});
+
+// Check for missing required fields in request bodies
+Object.entries(specs.paths).forEach(([path, methods]) => {
+  Object.entries(methods).forEach(([method, config]) => {
+    if (config.requestBody && config.requestBody.content) {
+      Object.values(config.requestBody.content).forEach(content => {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)